### DISCUSSION TITLE
Restricting Drag-and-Drop of Nodes  to Their Respective Sections

### DISCUSSION THREAD

**Discussion by adityasajoo** [ueberdosis/tiptap#4486] [General Questions]
I have an editor with blocks and sections, where "Block" and "Sections" are custom nodes. Each section contains several blocks. 
I've implemented drag-and-drop functionality for my editor using the data-drag-handle attribute. I want to ensure that blocks can only be dragged and dropped within their own sections and cannot be moved between sections.

![image](https://github.com/ueberdosis/tiptap/assets/61343444/895da6fa-1ce9-4fba-b7dd-bd4d8e7383af)

Here's the current setup:
- I have an editor with multiple custom sections.
- Each custom section contains custom Tiptap blocks.

I want to achieve the following behavior:
- Custom blocks in Section 1 should only be draggable within Section 1.
- Custom blocks in Section 2 should only be draggable within Section 2.
- Custom blocks should not be allowed to move between sections.

Any assistance would be greatly appreciated. Thank you!

**Comment by cchandurkar** [comment]
Did you find answer to that?
